homeassistant/components/homekit/strings.json

```diff
@@ -0,0 +1,54 @@
+{
+	"title" : "HomeKit Bridge",
```

Comment:
We should avoid 8 space indentation. It pushes the content too far to the right.

Reply:
I sent them though `python -m json.tool`

https://github.com/home-assistant/core/pull/35056